string unique_id
int16 acr_shelf
int16 action
bool status
bool failStatus